src/endpoints/stats/count_minted_domains.rs

```diff
@@ -0,0 +1,45 @@
+use crate::{models::AppState, utils::get_error};
+use axum::{
+    extract::{Query, State},
+    http::{HeaderMap, HeaderValue, StatusCode},
+    response::IntoResponse,
+    Json,
+};
+use axum_auto_routes::route;
+use mongodb::bson::{doc, Bson};
+use serde::{Deserialize, Serialize};
+use std::sync::Arc;
+
+#[derive(Serialize)]
+pub struct CountMintedDomainsData {
+    count: u64,
+}
+
+#[route(get, "/stats/count_minted_domains", crate::endpoints::stats::count_minted_domains)]
+pub async fn handler(
+    State(state): State<Arc<AppState>>,
+) -> impl IntoResponse {
+    let mut headers = HeaderMap::new();
+    headers.insert("Cache-Control", HeaderValue::from_static("max-age=60"));
+
+    let domain_collection = state
+        .starknetid_db
+        .collection::<mongodb::bson::Document>("domains");
+    let filter = doc! {
+        "creation_date": { "$gte": query.since },
```

Comment:
You didn't update so it doesn't compile...
Test your code!
`cargo run`
```rs
error[E0425]: cannot find value `query` in this scope
src/endpoints/stats/count_minted_domains.rs:29:36
   |
29 |         "creation_date": { "$gte": query.since },
   |                                    ^^^^^ help: a tuple struct with a similar name exists: `Query`
   ```